discord_bot/bot.py

```diff
@@ -8,58 +8,58 @@
 
 @interactions.listen()
 async def on_ready():
-  print("Ready")
-  print(f"This bot is owned by {bot.owner}")
+    print("Ready")
```

Comment:
Rebase your branch on to master and get the latest ruff rules. We have updated to use 2 spaces instead of 4. #21